lib/ahoy/intercom/store.rb

```diff
@@ -0,0 +1,41 @@
+require 'ahoy_matey'
+require 'intercom'
+
+module Ahoy
+  module Intercom
+    class Store < ::Ahoy::Stores::ActiveRecordStore
+
+      def intercom
+        @intercom ||= ::Intercom::Client.new(credentials)
+      end
+
+      def track_event(name, properties, options, &block)
+        super do |event|
+          create(event)
+        end
+      end
+
+      private
+
+      def create(event)
+        intercom.events.create(
+          event_name: event.name,
+          email: event.user.email,
+          user_id: event.user.id,
+          created_at: event.time.to_i,
+          metadata: event.properties
+        )
+      rescue ::Intercom::ResourceNotFound
+        #nothing do
```

Comment:
Am I right that here you rescue exception and don't even log it? What about other exceptions from intercom gem?